### PR TITLE
Fix safe subscript

### PR DESCRIPTION
**Pull request by hishbase**
Fix Collection safe subscript extension

### CODE REVIEW

File: ios/Source/Extensions/String+Core.swift [hunk 1]
```diff
@@ -24,6 +24,9 @@ extension String {
     /// Represents a forward slash
     public static let forwardSlash = "/"
 
+    /// Represents an @ sign
+    public static let atSign = "@"
```
Comment (johnnycoinbase) [inline ios/Source/Extensions/String+Core.swift]: nit - rename to just `at`?

File: ios/Source/Extensions/BidirectionalCollection+Core.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2020 Coinbase Inc. See LICENSE
+
+import Foundation
+
+extension BidirectionalCollection {
```
Comment (johnnycoinbase) [inline ios/Source/Extensions/BidirectionalCollection+Core.swift]: worth writing a few tests for this?

File: ios/Source/Extensions/String+Core.swift (hunk 1 above)
Comment (hishbase) [reply ios/Source/Extensions/String+Core.swift]: fixed

File: ios/Source/Extensions/BidirectionalCollection+Core.swift (hunk 2 above)
Comment (hishbase) [reply ios/Source/Extensions/BidirectionalCollection+Core.swift]: added

**Pull request merged**